english/wafflefries/wafflefries.md

```diff
@@ -0,0 +1,6 @@
+You sit down at the local bar and find some fries, the fries are given. 
+with a marshmellow sauce!?!?!
+
+[Dip the fry in the marshmellow?](../sleep/marshmallow.md)
+[Watch TV at the bar](www.youtube.com)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.